bitpay/client.py

```diff
@@ -46,9 +46,16 @@ def create_invoice(self, params):
       return response.json()['data']
     self.response_error(response)
 
-  def get_invoice(self, invoice_id): 
-    uri = self.uri + "/invoices/" + invoice_id
-    headers = {'accept': 'application/json'}
+  def get_invoice(self, invoice_id, token=None):
+    if token:
+      uri = self.uri + "/invoices/" + invoice_id + "?token=" + token
+      xidentity = key_utils.get_compressed_public_key_from_pem(self.pem)
+      xsignature = key_utils.sign(uri, self.pem)
+      headers = {"content-type": "application/json", 'accept': 'application/json', 'X-Identity': xidentity, 'X-Signature': xsignature, 'X-accept-version': '2.0.0'}
+    else:
+      uri = self.uri + "/invoices/" + invoice_id
+      headers = {"content-type": "application/json", 'accept': 'application/json'}
```

Comment:
sorry but, as mentioned, we're not accepting this addition

Reply:
Is that just for the public facade or for public and merchant facade?

For the merchant-facade I went by headers used in `create_invoice` and `verify_tokens`.

Reply:
we do not require of 'accept': 'application/json' on our API resource

Reply:
Uh... Now I'm completely confused...

> we do not require of 'accept': 'application/json' on our API resource
I didn't come up with the `'accept': 'application/json'` - that was introduced in #14 by someone else and it is used throughout the headers in this library despite the [API](https://bitpay.com/api/) not specifying it...

I'm really sorry for all of this back-and-forth - I'm not trying to get on your nerves... Just trying to create a PR which is up to your standards :)